### PR TITLE
Dotfiles Update

### PR DESCRIPTION
**Pull request by wrecker**
Removed the symlinks section.

### CODE REVIEW

File: bootstrap.json [hunk 1]
```diff
@@ -1,7 +1,6 @@
 {
     "after-scripts": [
-        "brew linkapps",
```
Comment (rygine) [inline bootstrap.json:3]: this is used to link the installed apps to the /Applications folder. i think we should keep this.

Comment (wrecker) [inline bootstrap.json:3]: its still there.. see below. Github's diff checker is not very smart. it did not detect that i removed the comma at the end of the line.

Comment (yanfali) [inline bootstrap.json:3]: use side by side to see this

**Comment by yanfali** [comment]
LGTM :+1:

**Comment by wrecker** [comment]
And from what we discussed yesterday we are not pruning any of the libs right?

**Pull request merged**